pkg/apis/config/settings/settings.go

```diff
@@ -42,6 +43,13 @@ type Settings struct {
 	BatchIdleDuration time.Duration
 }
 
+func (s Settings) Data() map[string]string {
```

Comment:
Does it make sense to leverage json serialization for this API, rather than hardcoding it? Check out our other APIs.